### DISCUSSION TITLE
Border for Modals

### DISCUSSION THREAD

**Discussion by newhinton** [Zettlr/Zettlr#2881] [General]
Hi, i could not find a way to add a border to the settings via css. Is it possible to do so, and maybe even with a radius?

I found that the white-on-white of the main window and the settings modal do not play well together, maybe it makes sense to add them as default

**Comment by nathanlesage** [comment]
The problem is that some Linux window managers simply don't apply a shadow. On Unity and (I think?) Gnome as well as macOS and Windows 7/10 the windows can be distinguished by this shadow.

If you don't see such a shadow, the way to go is to enable "use native appearance" in the Advanced preferences. This will introduce the default window Chrome around all of Zettlr's windows
